sdk/metrics/src/test/java/io/opentelemetry/sdk/metrics/testing/InMemoryMetricExporterTest.java

```diff
@@ -99,7 +108,7 @@ void testShutdown_export() {
   }
 
   @Test
-  void test_flush() {
+  void flush() {
```

Comment:
I normally try to stick with the test naming convention of the file I'm modifying, but in this case the `test_` prefix is purely redundant.